File: app/assets/javascripts/backbone/models/plane.js
var app = app || {};

// Model.
app.Plane = Backbone.Model.extend({
  urlRoot: function () {
    return '/planes/' + this.get('plane_id');
  },
  defaults: {
    name: "Plane 1",
    aisle: 5,
    seat: 6
  }
});

File: app/models/plane.rb
# == Schema Information
#
# Table name: planes
#
#  id         :integer          not null, primary key
#  name       :string
#  aisle      :integer
#  seat       :integer
#  created_at :datetime         not null
#  updated_at :datetime         not null
#

class Plane < ActiveRecord::Base

  belongs_to :flight
  validates :seat, :numericality => { :greater_than => 0}
  validates :seat, :numericality => { :less_than_or_equal_to => 6}
  validates :aisle, :numericality => {:greater_than => 0}
  validates :aisle, :numericality => { :less_than_or_equal_to => 20}
end
